t/file.t

```diff
@@ -23,20 +23,21 @@ if ( $^O =~ m/^MSWin/ ) {
   $tmp_dir = $ENV{TMP};
 }
 
-my $filename = "$cwd/test-$$.txt";
+my $filename = "$cwd/test-{a,b}-$$.txt";
```

Comment:
Yes when testing for the glob case we indirectly also test the array reference and the single string  but using it for the rest of the test would double the time <G>